rust/tests/fs_common/mod.rs

```diff
@@ -34,6 +34,7 @@ pub async fn create_table_from_json(
     assert!(path.starts_with("./tests/data"));
     std::fs::create_dir_all(path).unwrap();
     std::fs::remove_dir_all(path).unwrap();
+    std::fs::create_dir_all(path).unwrap();
```

Comment:
good question :D! Due to `Path` always canonicalizing all local path, they must exists - but I guess one crate_all is enough...

Reply:
turns out we needed the first crate all - remove all will panic if the folders do not exist (i.,e. in CI), but we still need empty folders, thus the second one. Added it back since it is "just" test code, but the proper solution would be to clean up directly after the tests.